app/classes/query/observations.rb

```diff
@@ -67,6 +67,15 @@ def model
     @model ||= Observation
   end
 
+  def alphabetical_by
+    @alphabetical_by ||= case params[:order_by].to_s
+                         when "user", "reverse_user"
+                           User[:login]
+                         when "name", "reverse_name"
+                           Name[:sort_name]
+                         end
+  end
+
```

Comment:
Even if we are restoring aplha sorting for Spl only, as I would suggest, we still DO need these lines restored.